x/programs/v2/test/programs/call_program/Cargo.toml

```
@@ -0,0 +1,14 @@
+[package]
+name = "call_program"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+wasmlanche-sdk = { path = "../../../../rust/wasmlanche-sdk" }
+borsh = { workspace = true }
+
+[build-dependencies]
+wasmlanche-sdk = { path = "../../../../rust/wasmlanche-sdk", features = ["build"] }
```

Pleeeeeaaaaase
```suggestion
wasmlanche-sdk = { path = "../../../../rust/wasmlanche-sdk", features = ["build"] }

```
I actually don't know if this suggestion adds a new line the way I think it would though 🤔